ui/src/views/Tasks/CreateTask/index.jsx

```diff
@@ -358,6 +422,17 @@ export default class CreateTask extends Component {
               </SpeedDial>
             </Fragment>
           )}
+          {dialogOpen && (
+            <DialogAction
+              open={dialogOpen}
+              title="Interactive Task"
+              body={<DialogInteractiveChangeText />}
+              onClose={() => this.props.history.replace('task/create')}
+              confirmText="Ok"
+              onComplete={() => this.setState({ dialogOpen: false })}
+              onSubmit={() => this.setState({ dialogOpen: false })}
```

Comment:
Instead of inlining the functions, let's create methods `handleDialogComplete`, `handleDialogSubmit`, `handleDialogClose`.